transactor/src/test/scala/io/mediachain/transactor/DummyContext.scala

```diff
@@ -40,3 +41,41 @@ object DummyContext {
     cleanupLogdir(context.logdir)
   }
 }
+
+case class DummyClusterContext(
+  dummies: Array[DummyContext]
+)
+
+object DummyClusterContext {
+  def setup(address1: String, address2: String, address3: String,
+            blocksize: Int = StateMachine.JournalBlockSize) = {
+    println("*** SETUP DUMMY COPYCAT CLUSTER")
+    val dummies = Array(address1, address2, address3)
+      .map { address =>
+        val store = new Dummies.DummyStore
+        val logdir = DummyContext.setupLogdir()
+        val server = Copycat.Server.build(address, logdir, store, blocksize)
+        val client = Copycat.Client.build()
+        DummyContext(server, client, store, logdir)
+    }
+    
+    // start cluster
+    dummies(0).server.bootstrap().join()
+    dummies(1).server.join(new Address(address1)).join()
```

Comment:
it's not at all confusing that these are two completely different `join`s, thanks java 😓